config/routes.rb

```diff
@@ -0,0 +1,24 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/merchants', to: 'merchants#index'
+
+  post '/merchants', to: 'merchants#create'
+  get '/merchants/new', to: 'merchants#new'
+
+  get '/merchants/:id/edit', to: 'merchants#edit'
+  patch '/merchants/:id', to: "merchants#update"
+
+  delete '/merchants/:id', to: 'merchants#delete'
```

Comment:
Rather than `delete`, name the controller action `destroy`.DELETE is the HTTP verb, and destroy is the CRUD action. They are easy to confuse. This route should be:

```
delete '/merchants/:id', to: 'merchants#destroy'
```